CMakeLists.txt

```diff
@@ -82,6 +78,8 @@ if (CMAKE_VERSION VERSION_LESS 3.1 OR NOT USE_COMPILE_FEATURES)
   endif ()
 endif ()
 
+set (CMAKE_CXX_FLAGS "-O3 -g -mavx -mfma ${CMAKE_CXX_FLAGS}")
+

```

Comment:
not portable at all!

Reply:
You're absolutely right. This is my problem; I didn't think it thoroughly. In our own fork, we already fixed this issue (https://github.com/resibots/libcmaes/tree/fix_flags_native) and I was planning to make a PR once #162 is merged..